homeassistant/components/ambient_station/__init__.py

```diff
@@ -176,6 +182,27 @@
     TYPE_LASTRAIN: ("Last Rain", None, TYPE_SENSOR, "timestamp"),
     TYPE_MAXDAILYGUST: ("Max Gust", SPEED_MILES_PER_HOUR, TYPE_SENSOR, None),
     TYPE_MONTHLYRAININ: ("Monthly Rain", "in", TYPE_SENSOR, None),
+    TYPE_PM25_24H: (
+        "PM25 24h Avg",
+        CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        TYPE_SENSOR,
+        None,
+    ),
+    TYPE_PM25_BATT: ("PM25 Battery", None, TYPE_BINARY_SENSOR, "battery"),
```

Comment:
Please use the appropriate device class constant. Only official device classes are allowed.
https://developers.home-assistant.io/docs/core/entity/binary-sensor#available-device-classes